src/main/resources/supporters.json

```diff
@@ -215,7 +215,7 @@
     "texture": "Volqonah",
     "type": 15,
     "status": "Temporary",
-    "bookId": "volqonah",
+    "bookId": "vampirism:valorous_tale",
```

Comment:
this is tricky, because the supporter file is uploaded and downloaded for older versions as well. This means that it must be backwards compatible. The mod gets the current supporter from [the api](https://api.vampirism.dev/api/v1/supporter/list). Therefor we can not use the same field for different ids.

We can name the new field just "book". In the code we then can ignore the "bookId" but in the supporter.json the "bookId" must be kept with the old values while adding the new key pairs as well.